docs/develop/java/observability.mdx

```diff
@@ -171,54 +171,78 @@ Here is how to query Workflow Executions:
 
 After you've created custom Search Attributes in your Temporal Service (using `temporal operator search-attribute create` or the Cloud UI), you can set the values of the custom Search Attributes when starting a Workflow.
```

Comment:
This isn't part of the original pull request but do you mind removing the line. "Here is how to query Workflow Executions:" I think this is part of the random typos from the migration and it's noticeable when you see on Vercel.